src/components/Sliders/Sliders.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { motion } from 'framer-motion';
+import First from './posters/1.jpg';
+import Second from './posters/2.jpg';
+import Third from './posters/3.jpg';
+import Fourth from './posters/4.jpg';
+import './Sliders.css';
+const Sliders = () => {
+  return (
+    <>
+      <div className="slider__container hidden lg:flex flex-row ">
+        <motion.div
+          animate={{
+            x: 79,
+            y: 16,
+          }}
+          transition={{ duration: 1 }}
```

Comment:
there is a glitch in the animation, it stops then continues. not sure what is the reason, maybe delay it a bit
`transition={{ duration: 1, delay: 1 }}`